### PR TITLE
Update to Chromium 61.0.3163.59

### PR DESCRIPTION
**Pull request by hferreiro**

### CODE REVIEW

File: atom/browser/atom_browser_client.h [hunk 1]
```diff
@@ -12,13 +12,19 @@
...
+#include "net/ssl/ssl_private_key.h"
+
 
 namespace content {
 class QuotaPermissionContext;
 class ClientCertificateDelegate;
 }
 
 namespace net {
+class ClientCertIdentity;
```
[the remark this answers is not in the document]
Comment (hferreiro) [reply atom/browser/atom_browser_client.h]: Actually, I just removed `net/ssl/client_cert_identity.h` there and added where really needed.

File: patches/master_patch.patch [hunk 2]
```diff
@@ -921,41 +951,22 @@ index c3f648f165483a2d618438c1a4ef1b35ad6fb10c..3c95e1256465b7b6a009d3bf7619c8f4
...
                     uint16_t items,
-diff --git a/components/guest_view/browser/guest_view_base.cc b/components/guest_view/browser/guest_view_base.cc
-index 27570202e8e5cca632e560fe5b22920aeb2c5742..3ac3b8c7b81c095b3b2264ff18431852de060a1b 100644
---- a/components/guest_view/browser/guest_view_base.cc
-+++ b/components/guest_view/browser/guest_view_base.cc
-@@ -432,8 +432,13 @@ void GuestViewBase::DidDetach() {
-   owner_web_contents()->Send(new GuestViewMsg_GuestDetached(
-       element_instance_id_));
-   element_instance_id_ = kInstanceIDNone;
--  if (!CanRunInDetachedState())
```
Comment (bridiver) [inline patches/master_patch.patch]: I don't think this is right. This is critical functionality for tab drag and drop cc @bbondy

File: patches/master_patch.patch [hunk 3]
```diff
@@ -1324,41 +1338,52 @@ index 829b2dc1c65e9c2d5226592b3b33c16ee4070b8b..a28adcf4478dddd5089271b7a4755b3b
...
 +                 print_pages_params_->params.document_cookie));
 +      }
        break;
+     case INVALID_SETTINGS:
++#if 0
+       Send(new PrintHostMsg_PrintPreviewInvalidPrinterSettings(routing_id(),
+                                                                cookie));
+-      print_preview_context_.Failed(false);
++#endif // 0
++      LOG(ERROR) << "PREVIEW FAILED.";
```
Comment (bridiver) [inline patches/master_patch.patch]: this looks like a dup of the code above?

File: patches/master_patch.patch (hunk 2 above)
Comment (ltilve) [reply patches/master_patch.patch]: Indeed. It was a chunk that slipped through the cracks, we have fixed that and will be updating the PR including it.

File: patches/master_patch.patch (hunk 3 above)
Comment (ltilve) [reply patches/master_patch.patch]: Sure. I'm also addressing it.

**Comment by hferreiro** [comment]
Continues here #286.

**Pull request closed without merging**